deployment/tfhub/tfhub-text-classification.ipynb

```diff
@@ -86,11 +86,12 @@
    "source": [
     "from verta import Client\n",
     "from verta.utils import ModelAPI\n",
+    "import os\n",
     "\n",
-    "# Use local env vars or uncomment and fill out the lines below:\n",
-    "# os.environ['VERTA_EMAIL'] = ''\n",
-    "# os.environ['VERTA_DEV_KEY'] = ''\n",
-    "# os.environ['VERTA_HOST'] = ''\n",
+    "# Ensure credentials are set up, if not, use below\n",
+    "# os.environ['VERTA_EMAIL'] = \n",
```

Comment:
I actually would leave the quotes here, because when you copy your dev key it doesn't include quotes.